src/main/java/fi/helsinki/cs/tmc/core/commands/VerifyCredentials.java

```diff
@@ -52,6 +62,10 @@ public Boolean call() throws TmcCoreException, IOException {
                 .makeGetRequest(URI.create(settings.getServerAddress() + TMC_SERVER_ROUTE), auth)
                 .getStatusCode();
 
-        return (response >= HTTP_SUCCESS_RANGE_MIN && response <= HTTP_SUCCESS_RANGE_MAX);
+        return isResponseInRange(response);
+    }
+
+    private boolean isResponseInRange(int response) {
```

Comment:
Consider renaming to `isStatusCodeSuccess()` or something similar, "in range" leaves much for interpretation.